sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Rules/ControlCharacterInString.cs

```diff
@@ -113,10 +113,16 @@ protected override void Initialize(SonarAnalysisContext context)
 
         private static void CheckControlCharacter(SyntaxNodeAnalysisContext c, string text, int displayPosIncrement)
         {
+            // simple verbatim - the node is StringLiteralExpression, the first token is verbatim
             if (c.Node.GetFirstToken().IsVerbatimStringLiteral())
             {
                 return;
             }
+            // interpolated verbatim - the node is InterpolatedStringText, the parent is InterpolatedStringExpression
+            if (c.Node.Parent.GetFirstToken().IsKind(SyntaxKind.InterpolatedVerbatimStringStartToken))
+            {
+                return;
+            }
```

Comment:
I think that extracting a method with the checks (e.g. `ShouldIgnore(SyntaxNode)`) will make the code a bit more readable. 

Also, instead of using comments I would create 2 methods `IsVerbatimString` and `IsInterpolatedVerbatimString`.